unit2-java/lesson16.rst

```diff
@@ -5,24 +5,19 @@ Goals
 -----
 
 - Understand how object-relational mapping allows object-oriented web applications to store data in relational databases
-- Implement ORM with one-to-one and one-to-many relationships in Spring Boot
+- Implement ORM for individual, non-related entities in Spring Boot
 
 Objectives
 ----------
 
 - Explain what object-relational mapping refers to
 - Describe what a data layer is and how it is used in ORM
-- Describe the one-to-many relationship
 - Use the following annotations and describe what they accomplish:
```

Comment:
This sentence is rendered as bold and italic. I think you need to add an empty line between it and the sub-list.